android/variables.gradle

```diff
@@ -1,5 +1,5 @@
 ext {
-    minSdkVersion = 23
+    minSdkVersion = 24
```

Comment:
Please revert this change. The `minSdkVersion` for Capacitor 7 is 23.

Reply:
Yes but the Android implementation itself requires API 24 https://developers.google.com/ml-kit/vision/subject-segmentation/android

Reply:
Okay, please add this to the plugin readme.